### PR TITLE
On Home Page, Sign In Page

### PR DESCRIPTION
**Pull request by oleksandranikitiuk**

### CODE REVIEW

**Review by derkachOksana — DISMISSED** [review]

File: src/test/java/pages/HomePage.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    public void clickSignUpBtn() {
+        signUpBtn.click();
+    }
+    public void clickRestaurantsListBtn() {
+        restaurantsListBtn.click();
+    }
+    public void clickViewAllTag() {
+        viewAllTag.click();
+    }
+
```
Comment (dmatsev) [inline src/test/java/pages/HomePage.java]: please remove empty lines

File: src/test/java/pages/SignInPage.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    public void setUserPasswordInputField(String password) {
+        userPasswordInputField.sendKeys(password);
+    }
+
+    public void clickSignInBtn() {
+        signInBtn.click();
+    }
+
+    public void getHomeBtn() {
```
Comment (dmatsev) [inline src/test/java/pages/SignInPage.java]: Please rename this method corresponding to its implementation.

**Review by dmatsev — DISMISSED** [review]
Please resolve the comments.

File: src/test/java/pages/HomePage.java (hunk 1 above)
Comment (oleksandranikitiuk) [reply src/test/java/pages/HomePage.java]: Done

File: src/test/java/pages/SignInPage.java (hunk 2 above)
Comment (oleksandranikitiuk) [reply src/test/java/pages/SignInPage.java]: Done

**Review by derkachOksana — APPROVED** [review]

**Review by dmatsev — APPROVED** [review]

**Pull request merged**